homeworks/html-css/Fenestro/index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="container">
+        <div class="top">
+            <div class="city">New York, NY</div>
+            <div class="todayDate">
+                Tuesday, April 15th <br>
+                Overcast
+            </div>
+            <div class="content">
+                <div class="weatherImg">
+                    <img src="./icons/png/iconfinder_weather04_4102315.png" alt="">
+                </div>
+                <div class="degree">
+                    <span class="deg">58</span>
+                    <span class="small">&#8457</span>
+                </div>
+                <div class="details">
+                    <div class="item">Precipitation: 100%</div>
+                    <div class="item">Humidity: 97%</div>
+                    <div class="item">Wind 4 mph SW</div>
+                    <div class="item">Pollen Count: 36</div>
+                </div>
+            </div>
+        </div>
+        
+        <div class="dayContainer">
+            <div class="day">
```

Comment:
Use  more descriptive class names for all project

https://habr.com/ru/post/303174/
https://github.com/yoksel/common-words